results.css

```diff
@@ -0,0 +1,34 @@
+@import url(index.css);
+@import url(countries.css);
+
+.heading {
+    color: var(--secondary-color);
+}
+
+.sub-heading {
+    color: var(--secondary-color);
+}
+
+.correct-answer {
+    background-color: #22B14C !important;
+}
+.your-answer {
+    background-color: #FF6961 !important;

```

Comment:
Using !important is a bad practice. If the property (`background-color` in this case) is not getting applied, solve it with specificity. Avoid using !important.

You can read more about it here: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception